lib/driver/base.js

```diff
@@ -257,8 +258,8 @@ module.exports = Base = Class.extend({
     }
   },
 
-  addMigrationRecord: function (name, callback) {
-    this.runSql('INSERT INTO "' + global.migrationTable + '" (name, run_on) VALUES (?, ?)', [name, new Date()], callback);
+  addMigrationRecord: function (name, migrationTable, callback) {
+    this.runSql('INSERT INTO "' + migrationTable + '" (name, run_on) VALUES (?, ?)', [name, new Date()], callback);
```

Comment:
Everywhere, its already global no need to pass it around ;)

Am 10.02.2015 um 21:40 schrieb Tom Caflisch:
>
> In lib/driver/base.js
> <https://github.com/kunklejr/node-db-migrate/pull/217#discussion_r24448394>:
>
> > @@ -257,8 +258,8 @@ module.exports = Base = Class.extend({
> >      }
> >    },
> >  
> > -  addMigrationRecord: function (name, callback) {
> > -    this.runSql('INSERT INTO "' + global.migrationTable + '" (name, run_on) VALUES (?, ?)', [name, new Date()], callback);
> > +  addMigrationRecord: function (name, migrationTable, callback) {
> > +    this.runSql('INSERT INTO "' + migrationTable + '" (name, run_on) VALUES (?, ?)', [name, new Date()], callback);
>
> Do you want global.migrationTable everywhere that I use it instead of
> passing a param? Or just in the base.js?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kunklejr/node-db-migrate/pull/217/files#r24448394>.
>